libgfortran/libgfortran.h

```diff
@@ -67,8 +67,14 @@ extern float __strtof (const char *, char **);
 #define gfc_strtof __strtof
 extern double __strtod (const char *, char **);
 #define gfc_strtod __strtod
+
 extern long double __strtold (const char *, char **);
+#if defined(__arm__) || defined(__aarch64__) || defined(_ARM_) || defined(_ARM64_)
```

Comment:
This fix seems to be unrelated to the other one (which is equivalent to [https://github.com/ZacWalk/gcc-woarm64/pull/5/files).](https://github.com/ZacWalk/gcc-woarm64/pull/5/files).) What was the symptoms, e.g. error message, of the issue this change is fixing?

Reply:
Moved to a separate PR
https://github.com/ZacWalk/gcc-woarm64/pull/10